Website/Portals/_default/HotcakesViews/_default/Scripts/Checkout.js

```diff
@@ -42,12 +42,6 @@
                 behavior: 'smooth'
             });
         });
-
-        $("#btnsubmit").click(function (e) {
```

Comment:
@VAlmea Is this supposed to be removed entirely, or should the `btnSubmit` be updated to match the `hcTakeOrder` class?

Reply:
No. It must be removed. It was just a test.